### PR TITLE
Adding ultrasound phantom dataset

### PR DESCRIPTION
**Pull request by Ouwen**
This dataset contains ultrasound phantoms used in the following paper: https://arxiv.org/abs/1908.05782.

The license is https://creativecommons.org/licenses/by-nc/4.0/

### CODE REVIEW

**Comment by Ouwen** [comment]
@mlp6

**Comment by mlp6** [comment]
@Ouwen is this file needed: https://github.com/tensorflow/datasets/pull/1027/commits/2b1ff486fb684a2dab01788a71845d18925d2627#diff-f970146f264b6cf0a5614fbc0499c65a

**Comment by mlp6** [comment]
@Ouwen should also add acknowledgement to NIH funding in `setup.py` and/or other equivalent files.

**Comment by Ouwen** [comment]
> @Ouwen is this file needed: [2b1ff48#diff-f970146f264b6cf0a5614fbc0499c65a](https://github.com/tensorflow/datasets/commit/2b1ff486fb684a2dab01788a71845d18925d2627#diff-f970146f264b6cf0a5614fbc0499c65a)

Yeah these checksums will change when we switch over to duke library data hosting.

File: tensorflow_datasets/image/duke_ultrasound.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+reduction, contrast enhancement and improvement in conspicuity of 
+anatomical structures."""
+
+_URLS = ['https://arxiv.org/abs/1908.05782', 'https://github.com/ouwen/mimicknet']
+
+
+class DukeUltrasound(tfds.core.GeneratorBasedBuilder):
+  """DAS beamformed phantom images and paired post-processed images."""
+
+  VERSION = tfds.core.Version("1.0.0",
```
Comment (us) [inline tensorflow_datasets/image/duke_ultrasound.py]: Why false ? you don't have to add old version.

File: tensorflow_datasets/image/duke_ultrasound.py [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+                              experiments={tfds.core.Experiment.S3: False})
+  SUPPORTED_VERSIONS = [
+      tfds.core.Version("3.0.0", "S3: www.tensorflow.org/datasets/splits"),
```
Comment (us) [inline tensorflow_datasets/image/duke_ultrasound.py]: you should first add version 2.0.0

**Comment by Ouwen** [comment]
@us please let me know if this PR is good for merging.

**Comment by Ouwen** [comment]
@Conchylicultor we are the original creators of this dataset, please let me know how we can get this in the tfds official GCS bucket.

**Comment by Ouwen** [comment]
@us I just wanted to give you a ping regarding this PR.

**Pull request merged**